gradle.properties

```diff
@@ -25,6 +25,8 @@ kotlin.native.ignoreDisabledTargets=true
 systemProp.kotlin.native.disableCompilerDaemon=true
 org.gradle.configureondemand=false
 kotlin.mpp.stability.nowarn=true
+# https://youtrack.jetbrains.com/issue/KT-58063 (should be fixed in Kotlin 1.9)
+org.gradle.configuration-cache=false
```

Comment:
Configuration cache is now stable, so it's reasonable that people will enable it in `~/.gradle/gradle.properties`. Unfortunately, some multiplatform tasks are not yet compatible with it